src/Appwrite/Auth/OAuth2/Etsy.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+namespace Appwrite\Auth\OAuth2;
+
+use Appwrite\Auth\OAuth2;
+
+class Etsy extends OAuth2
+{
+    /**
+     * @var string
+     */
+    private $endpoint = 'https://api.etsy.com/v3/public';
```

Comment:
We have to add typings here and everywhere else 👍🏻 
```suggestion
    private string $endpoint = 'https://api.etsy.com/v3/public';
```